content/tutorials/generic/micropython-installation/micropython-installation.md

```diff
@@ -0,0 +1,205 @@
+
+### Finding BOSSAC
+
+In this step you will be diving into the core files you downloaded in the first step. This process will be different for you depending on if you're on a Windows or MacOS computer.
+
+#### Windows
+
+The file you will need to access can be found in 
+```
+C:\Users\{your-username}\AppData\Local\Arduino15\staging\packages\bossac-1.9.1-arduino2-windows.tar.gz\bossac-1.9.1-arduino2-windows.tar\bin\` 
+```
+
+and is named `bossac.exe`.
+
+One you've found the file, extract it from the .tar archive and copy it somewhere else, your desktop for example. 
+
+Open a command terminal and start typing in `start `, now drag and drop the bossac.exe file you've found into the terminal, and press enter.
+
+If successful, another command terminal window should open in which you are able to execute the command that will flash your board with the MicroPython firmware.
+
+Execute the following command, but replace the port with the name of the port where your board is connected, and the firmware file with the directory where you have placed the firmware you downloaded previously:
+
+```
+bossac -e -w --offset=0x16000 --port={port} -i -d -U -R {firmware}
+```
+
+Again, don't disconnect or power off your board during this part of the process, watch the progress of flashing the firmware to your board. Once complete, you're all set and you're ready to start programming the board in MicroPython.
+
+Go to the Arduino Lab MicroPython IDE and press connect in the top left, choose the port, and code away!
+
+#### MacOS
+
+The file you will need to access can be found in 
+```
+Users/{your-user}/Library/Arduino15/packages/arduino/tools/bossac/1.9.1-arduino2
+```
+ the file you are after is `bossac.exe`
+
+>**Note:** The `"Library"` directory is hidden, press `Shift + Command + . ` to reveal hidden directories. 
+
+Open a terminal window and drag and drop the bossac.exe file into the terminal window.
+
+Now, you should be able to execute the command that will flash your board with the MicroPython firmware.
+
+Execute the following command, but replace the port with the name of the port where your board is connected, and the firmware file with the directory where you have placed the firmware you downloaded previously:
+
+```
+bossac -e -w --offset=0x16000 --port={port} -i -d -U -R {firmware}
+```
+
+Again, don't disconnect or power off your board during this part of the process, watch the progress of flashing the firmware to your board. Once complete, you're all set and you're ready to start programming the board in MicroPython.
+
+Go to the Arduino Lab MicroPython IDE and press connect in the top left, choose the port, and code away!
+
+## Arduino Nano RP2040 Connect
+
+If you want to program your Arduino board using MicroPython, and have an Arduino Nano RP2040 to use for this purpose, congratulations! You have chosen the easiest board to flash with the MicroPython firmware!
+
+All you need to do is to:
```

Comment:
To program your Arduino Nano RP2040 board with MicroPython, follow the instructions just below. The Nano RP2040 is the easiest board to get started with MicroPython, as the installation requires no additional software tools for installation.